content/docs/tutorials/moneygram-access-integration-guide.mdx

```diff
@@ -0,0 +1,390 @@
+---
+title: Integrate with MoneyGram Access
+order: 140
+---
+
+import { CodeExample } from "components/CodeExample";
+import { Alert } from "components/Alert";
+
+This document guides the reader through the technical requirements for integrating [MoneyGram Access] into an existing application. MoneyGram Access is a MoneyGram product that enables users of third-party applications, such as crypto wallets and exchanges, to cash-in (deposit) and cash-out (withdrawal) of Stellar USDC.
+
+<Alert>
+
+
+This document does not describe the business requirements for integrating with MoneyGram Access.
+
+</Alert>
+
+
+## Resources
+
+- [MoneyGram Access Wallet MVP Implementation]
+  - Use this MVP implementation as a reference for building your own integration. Many of the code snippets shared in this document are pulled from this project.
+- [Stellar Test Anchor]
+  - Before getting access to MoneyGram's test environment, you can use the SDF's test anchor while developing your integration
+- [Stellar Demo Wallet]
+  - This application visualizes the API calls necessary to connect to a Stellar Anchor
+- [Stellar Ecosystem Proposal 24 (SEP-24)]
```

Comment:
You can use the `[text][anchor alias]` syntax in Markdown to avoid repeating links in the footnote, e.g. `[Stellar Ecosystem Proposal 24 (SEP-24)][sep-24]` and have just one `[sep-24]: https://stellar.org/protocol/sep-24` definition at the end.